framework/resources/Functional/hive/hive_storage/skipHeaderFooter/hiveHeaderFooter.json

```diff
@@ -18,10 +18,15 @@
         }
     ],
     "datasources": [
-    { 
-       "mode": "gen", 
-       "src": "Datasources/hive/execHive.sh framework/resources/Datasources/hive_storage/skipHeaderFooter/skipHeaderFooter.ddl", 
-       "dest": "" 
-    }
+    {
+       "mode": "gen",
+       "src": "Datasources/hive/execHive.sh framework/resources/Datasources/hive_storage/skipHeaderFooter/skipHeaderFooter.ddl",
+       "dest": ""
+    },
+      {
+        "mode": "gen",
+        "src": "Datasources/hive_storage/skipHeaderFooter/DRILL_5941_prepare_data.sh",
+        "dest": ""
+      }
```

Comment:
Can you please fix the formatting (the two gen sections are not aligned)

Reply:
Done.